src/app/content/studyGuides/components/Filters.tsx

```diff
@@ -31,13 +34,48 @@ const ConnectedFilterList = connect(
   })
 )(FiltersList);
 
-export default () =>
-  <Filters>
+// tslint:disable-next-line:variable-name
+const ConnectedPrintButton = connect(
+  (state: AppState) => ({
+    disabled: selectors.summaryIsLoading(state),
+    loading: selectors.summaryIsLoading(state),
+    shouldFetchMore: selectors.hasMoreResults(state),
+  }),
+  (dispatch: Dispatch) => ({
+    loadHighlightsAndPrint: flow(printStudyGuides, dispatch),
+  }),
+  (stateProps, dispatchProps, ownProps: {onClick: () => void}) => {
+    const {shouldFetchMore, loadHighlightsAndPrint, ...props} = {
+      ...stateProps,
+      ...dispatchProps,
+      ...ownProps,
+    };
+
+    const onClick = () => {
+      ownProps.onClick();
+
+      if (shouldFetchMore) {
+        loadHighlightsAndPrint();
```

Comment:
you can follow the same pattern as `src/app/content/highlights/components/SummaryPopup/Filters.tsx` and only bind the onClick that dispatches `loadHighlightsAndPrint` if there are more results, if no onClick is specified the button will open print dialog itself

Reply:
Should there be no separate analytics event for printing study guides? Just use the one that listens to any prints?

Reply:
i don't think there is a custom event for this, we probably just rely on the button click event

Reply:
I've removed the custom event i added if that's what you meant.